docs/in-depth/calling-momemta.md

```diff
@@ -18,24 +20,60 @@ Configuration my_config = my_reader.freeze();
 
-!!! tip ""
-    The phase-space points are associated with a weight, given that they are not distributed uniformly in the phase-space, but according to the importance function defined by Vegas. In most use cases this weight is not needed by the user since Cuba automatically takes it into account when computing the integral, however it can still be retrieved through the `cuba::ps_weight` input tag (see [here](configuration-file.md#defaults)).
+The function `computeWeights()` starts the Monte-Carlo integration: the integrand function is called a large number of times, each time passing as input a phase-space point vector (where the length of the vector is the dimensionality of the integrated phase-space) with elements between 0 and 1, and returning as output the integrated function evaluated on this point.
+
+`computeWeights()` returns a vector of pairs (weight, uncertainty). In most cases, this vector will contain only one entry, but MoMEMta allows the possibility to define *vector* integrands, i.e. integrate multi-valued functions, and return a weight and uncertainty for each component.
+
+[//]: # (> *The phase-space points are associated with a weight, given that they are not distributed uniformly in the phase-space, but according to the importance function defined by Vegas. In most use cases this weight is not needed by the user since Cuba automatically takes it into account when computing the integral, however it can still be retrieved through the `cuba::ps_weight` input tag (see [here](configuration-file.md#defaults)).*)
+
+## Python bindings
+
+Calling MoMEMta from python is also possible, provided the python bindings have been built (see the [build options](../getting-started.md#build-options)). If you have not installed MoMEMta system-wide but in a user-defined directory (say `MOMEMTA_DIR`), you should make sure the two following conditions are satisfied:
+
+  * `$MOMEMTA_DIR/lib64/pythonX/site-packages/` should be in your `PYTHONPATH` (`X` might change depending on how you built MoMEMta)
+  * `$MOMEMTA_DIR/lib/` should be in your `LD_LIBRARY_PATH`
+
+Then, you might call MoMEMta from your python analysis script in this manner:
+```python
+import momemta
+import ROOT
+
+# Restrict logging to "error" level
+momemta.set_log_level(momemta.log_level.error)
+
+momemta_cfg = momemta.ConfigurationReader("path_to_config.lua")
+momemta_computer = momemta.MoMEMta(momemta_cfg.freeze())
+
+# Much shorter this way!
+LorentzVector = ROOT.Math.LorentzVector(ROOT.Math.PxPyPzE4D("double"))
+
+p1 = LorentzVector(10, 20, 30, 200)
+p2 = LorentzVector(-10, 20, -30, 200)
+
+result = momemta_computer.computeWeights([p1, p2])
+print("Weight: {} +- {}".format(result[0][0], result[0][1]))
+```
```

Comment:
I'm not sure, but maybe we can also state here that you can pass simple array instead of LorentzVector, like `[px, py, pz, E]`. In fact, if `ROOT` is not built with python support, that's the only way to go